firmware/serial_debugger/board_protos/VC.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+/*
+For enums, the default value is the first defined enum value, which must be 0.
+For numeric types, the default value is zero.
+*/
+
+enum GpioNetName {
+  GPIO_NET_NAME_UNSPECIFIED  = 0;
```

Comment:
did you get a chance to test every gpio and adc pin with the VC and verify it works?

Reply:
No I haven't tested each one, just a few of the new GPIO and ADC pins. Figured this would be fine?

Reply:
I will do some more testing today